app.js

```diff
@@ -16,46 +14,53 @@ const incorrect = document.getElementById('incorrect');
 const correct = document.getElementById('correct');
 const total = document.getElementById('total');
 
-// let state
 let ballLocation;
 let clicked;
+let ball;
 let scoreCorrect = 0;
 let scoreIncorrect = 0;
 let scoreTotal;
 const cupArray = [cup1, cup2, cup3];
 
-// set event listeners 
-  // get user input
-  // use user input to update state 
-  // update DOM to reflect the new state
-
 button1.addEventListener('click', () => {
     clicked = cup1;
+    ball = ball1;
     getRandomItem(cupArray);
     revealReset();
-    // score();
     handleGuess(clicked, ballLocation);
-    console.log(clicked);
 });
 
 button2.addEventListener('click', () => {
     clicked = cup2;
+    ball = ball2;
     getRandomItem(cupArray);
     revealReset();
-    // score();
     handleGuess(clicked, ballLocation);
 });
 
 button3.addEventListener('click', () => {
     clicked = cup3;
+    ball = ball3;
     getRandomItem(cupArray);
     revealReset();
-    // score();
     handleGuess(clicked, ballLocation);
 });
 
+function cupAnimation() {
+    clicked.classList.add('cup-animation');
+    setTimeout(() => {
+        clicked.classList.remove('cup-animation');
+    }, 500);
+}
+
+function ballAnimation() {
+    ball.classList.add('ball-animation');
+    setTimeout(() => {
+        ball.classList.remove('ball-animation');
+    }, 500);
+}
```

Comment:
I love how you handled the cupAnimation/ballAnimation functions.  This works really well from a user's perspective and it is easy to follow in your code.